tests/src/TestCase.php

```diff
@@ -49,12 +51,7 @@ protected function getPackageProviders($app): array
         ];
     }
 
-    protected function defineDatabaseMigrations(): void
-    {
-        $this->loadMigrationsFrom(__DIR__ . '/../database/migrations');
```

Comment:
Using this causes Laravel to migrate the database twice, using `RefreshDatabase` trait (which starts a database transaction) and explicitly via `loadMigrationsFrom()`. 

Using `testbench.yaml` configurations only registers the paths to `Migrator` and runs the migration once via `RefreshDatabase` eliminating the issue introduced by caching PDO instances between tests: https://github.com/laravel/framework/pull/47912